src/SDKs/IotCentral/Management.IotCentral/Microsoft.Azure.Management.IotCentral.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.IotCentral</PackageId>
+		<Description>Provides management capabilities for Microsoft Azure IotCentral.</Description>
+		<AssemblyTitle>Microsoft Azure IotCentral Management</AssemblyTitle>
+		<AssemblyName>Microsoft.Azure.Management.IotCentral</AssemblyName>
+		<Version>1.0.0-preview</Version>
+		<PackageTags>Microsoft Azure IotCentral;IotCentral management;IotCentral;</PackageTags>
+		<PackageReleaseNotes>
+			<![CDATA[
+        This is a public preview release of the Azure IoT Central SDK.
+      ]]>
+		</PackageReleaseNotes>
```

Comment:
nit: indentation is usually 2 spaces.

Reply:
Fixed, there was a mix of tabs and spaces here.